sdk/src/main/java/com/akkaserverless/javasdk/lowlevel/ValueEntityHandler.java

```diff
@@ -37,8 +37,8 @@
    * @param context The command context.
    * @return The reply to the command, if the command isn't being forwarded elsewhere.
    */
-  ValueEntityBase.Effect<Any> handleCommand(Any command, Any state, CommandContext<Any> context)
-      throws Throwable;
+  ValueEntityBase.Effect<? extends GeneratedMessageV3> handleCommand(
+      Any command, Any state, CommandContext<Any> context) throws Throwable;
```

Comment:
this is a leftover from previous PR #73 

I will move this to separate PR later, should not be here.